inji-web/src/test-utils/reduxMockUtils.tsx

```diff
@@ -0,0 +1,19 @@
+// reduxMockUtils.ts
```

Comment:
Why do we need a separate mockUtil for redux, lets consume the existing mockUtil

Reply:
Added  todo comment.